### ISSUE TITLE
documentation: manual doesn't describe trace format

### ISSUE THREAD

**Issue by yarden** [Kappa-Dev/KaSim#601]
As far as I can tell, the trace file (generated by ``-trace`` option to KaSim) isn't documented anywhere. Traces don't appear at all in the latest Kappa [PDF manual](https://kappalanguage.org/sites/kappalanguage.org/files/inline-files/Kappa_Manual.pdf) and the format isn't described anywhere else (including in the [Trace Query Language page](https://www.cs.cmu.edu/~jlaurent/software/katql-documentation.html)).

**Comment by hmedina** [comment]
Indeed! I'll just add that the TQL uses the KaSim parser, so it itself doesn't know (or care) about the trace format. Only KaSim knows what's in that gargantuan one-liner.

**Comment by pirbo** [comment]
So... there is an (hidden) attempt of documentation at https://tools.kappalanguage.org/kappa-trace-checker/trace-format-description.html but yes it need to be improved join a visible place!

**Comment by yarden** [comment]
Thanks @pirbo. A quick clarification: conversations with Jonathan suggested that the trace format is determined semi-automatically by serialization libraries used by KaSim - meaning it doesn't necessarily have a fixed format. Is that no longer true? Just trying to understand how the trace format was/is determined.

**Comment by pirbo** [comment]
The serialization library for reading and writing the trace used in Kappa software does not generate automatically the format of the trace. We have control on the scheme of the JSON it generates.
That does not mean that I don't dislike this quickly and dirtily determined format and therefore that I change it on occasion to go toward what I think it should be :-).
If your underlying question is : can I rely on the trace format to write my analyses? This is a good question and we should discuss what you need and what you're trying to compute in order to determine if you live in a satisfactory and therefore stable subset of the trace schema or not... :-)
